pom.xml

```diff
@@ -126,6 +126,7 @@
         <spring-security.version>5.3.10.RELEASE</spring-security.version>
         <grpc.version>1.33.1</grpc.version>
         <rocketmq-client.version>4.9.3</rocketmq-client.version>
+        <pulsar-client.version>2.10.1</pulsar-client.version>
```

Comment:
add license in  shenyu/shenyu-bootstarp-dis/license file

Reply:
> add license in shenyu/shenyu-bootstarp-dis/license file

fixed